src/tfrnnlmbin/lattice-lmrescore-tf-rnnlm.cc

```diff
@@ -0,0 +1,142 @@
+// tfrnnlmbin/lattice-lmrescore-tf-rnnlm.cc
+
```

Comment:
are you checking that this is compiled from the scripts and giving a suitable message if not?  I don't want to have users asking about this on the list.